### DISCUSSION TITLE
Month Picker

### DISCUSSION THREAD

**Discussion by half5cat** [carbon-design-system/carbon#8164] [Help] [answered]
Can the date picker be modified in order to become a month picker or is the only option to use the simple date input?

I am looking for something similar to this:
![image](https://user-images.githubusercontent.com/1887866/112128676-f1b6b300-8bc6-11eb-8f68-6984528db707.png)

**Comment by vpicone** [comment]
I think a regular old dropdown is probably the expected/most straight forward UX here.

**Comment by jnm2377** [comment, marked as answer]
But to answer your question, specifically, we don't have any variation of date picker that would allow you to just select months.

**Reply by half5cat** [reply]
Alright, thank you, @jnm2377 !
